docs/about-core-concepts/metadata-types.rst

```diff
@@ -8,4 +8,4 @@ Metadata Types
 
     Metadata type yaml file must contain name, description and dataset keys.
 
-    Dataset key must contain id, sources, creation_dt, label and search_fields keys.
+    Dataset key must contain id, sources, grid_spatial, measurements, creation_dt, label, format, and search_fields keys.
```

Comment:
This sentence is not strictly true currently.  ODC 1.8.x allows for non-spatial datasets (e.g. for storing telemetry metadata), which can be searched for but can't be loaded into an xarray with spatial dimensions.  The metadata types for these do not have to have grid_spatial, measurements and format.

But for metadata types for spatial datasets that can be loaded into xarrays with spatial dimensions, this sentence is true.

I'm leaning towards dropping support for non-spatial metadata types/products/datasets in ODCv2, but they will still be at least partially supported into v1.9

Reply:
Is the updated wording better?